bambam.6

```diff
@@ -29,7 +29,13 @@ colours.
 The screen is cleared at random.
 .SH NOTES
 \fBTo quit, type quit in the game\fR.
+.PP
 To turn the sound off and on, type mute and unmute, respectively, in the game.
+.PP
+Images (gif) and sounds (wav) located in \fB$XDG_DATA_HOME\fR (usually
+~/.local/share/bambam/data) will be loaded in addition of distribution
```

Comment:
Please change the ending to "in addition to resources distributed with the game."

Reply:
Done.